Makefile

```diff
@@ -278,6 +278,10 @@ ui-provider-build:
 ui-integration-tests: ui-setup
 	cd ui; npm run ci-test-integration; cd ..
 
+## rtk query code generation
+rtk-query-codegen:
```

Comment:
Will you put this in a different PR? This needs to be added to the list of dependency checks and we need to make sure that we’re actually publishing this package.

Reply:
got accidentaly pushed .